VENDOR_NEUTRALITY.md

```diff
@@ -0,0 +1,33 @@
+# Metal3-io vendor neutrality declaration
+
+As a Cloud Native Computing Foundation project, Metal3-io will keep its
+[vendor neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/)
+
+**Hardware vendor neutrality:**
+As it is stated in the [documentation](https://book.metal3.io/) Metal3-io is a
+CNCF project with a goal to provide bare metal life cycle management
+functionality regardless of the vendor of the  hardware. The only requirement
+is that the machines are equipped with a [BMC](https://www.servethehome.com/explaining-the-baseboard-management-controller-or-bmc-in-servers/)
+and support either [Redfish](https://redfish.dmtf.org/) or [IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface)
+protocols. Metal3 also supports vendor specific versions of the aforementioned
+protocols but that support is on top of the standard protocols and the reason
+is to provide full hardware compatibility  across the widest range of vendors
+possible.
+
+**Feature vendor neutrality:**
+Metal3 features are developed with the specific aim to manage the life cycle
+of bare metal machines and  K8s clusters on top of those machines thus the
+Metal3  stack by nature only supports so called "self-hosted" scenarios,
+none of the Metal3's features require any sort of commercial infrastructure
+provider service (e.g. AWS, Azure etc...) to be usable. In regards to feature
```

Comment:
Would you please break this big sentence in two for better readability?

Reply:
done thanks